contracts/Aavegotchi/ForgeDiamond/facets/ForgeDAOFacet.sol

```diff
@@ -133,35 +135,47 @@ contract ForgeDAOFacet is Modifiers {
         return s.alloyBurnFeeInBips;
     }
 
-    //    // @notice Allow DAO to update percent chance to win from a Geode.
-    //    // @param points RarityValueIO struct of points
-    //    function setGeodeWinChance(RarityValueIO calldata chances) external onlyDaoOrOwner {
-    //        s.geodeWinChance[COMMON_RSM] = chances.common;
-    //        s.geodeWinChance[UNCOMMON_RSM] = chances.uncommon;
-    //        s.geodeWinChance[RARE_RSM] = chances.rare;
-    //        s.geodeWinChance[LEGENDARY_RSM] = chances.legendary;
-    //        s.geodeWinChance[MYTHICAL_RSM] = chances.mythical;
-    //        s.geodeWinChance[GODLIKE_RSM] = chances.godlike;
-    //
-    //        emit SetGeodeWinChance(chances);
-    //    }
-    //
-    //    // @notice Allow DAO to set which prizes can be won from a Geode.
-    //    // @param ids Token IDs of the available prizes
-    //    // @param quantities Initial amounts of each prize available
-    //    function setGeodePrizes(uint256[] calldata ids, uint256[] calldata quantities) external onlyDaoOrOwner {
-    //        require(ids.length == quantities.length, "ForgeDAOFacet: mismatched arrays");
-    //
-    //        for (uint256 i; i < ids.length; i++){
-    //            if (s.geodePrizeQuantities[ids[i]] == 0){
-    //                // this ID is deleted from the array in the geode opening function when last item is won.
-    //                s.geodePrizeTokenIds.push(ids[i]);
-    //            }
-    //            s.geodePrizeQuantities[ids[i]] += quantities[i];
-    //        }
-    //
-    //        emit SetGeodePrizes(ids, quantities);
-    //    }
+    // @notice Allow DAO to update percent chance to win from a Geode.
+    // @param points RarityValueIO struct of points
+    function setGeodeWinChanceBips(RarityValueIO calldata chances) external onlyDaoOrOwner {
+        s.geodeWinChanceBips[COMMON_RSM] = chances.common;
+        s.geodeWinChanceBips[UNCOMMON_RSM] = chances.uncommon;
+        s.geodeWinChanceBips[RARE_RSM] = chances.rare;
+        s.geodeWinChanceBips[LEGENDARY_RSM] = chances.legendary;
+        s.geodeWinChanceBips[MYTHICAL_RSM] = chances.mythical;
+        s.geodeWinChanceBips[GODLIKE_RSM] = chances.godlike;
+
+        emit SetGeodeWinChance(chances);
+    }
+
+    // @notice Allow DAO to set which prizes can be won from a Geode.
+    // @param ids Token IDs of the available prizes
+    // @param quantities Initial amounts of each prize available
+    function setGeodePrizes(uint256[] calldata ids, uint256[] calldata quantities) external onlyDaoOrOwner {
```

Comment:
Shouldn't this function also transfer prizes from msg.sender into the Forge diamond? @infiniRekt

Reply:
That's what we do with the raffle contract